std_execution.bs

```diff
@@ -5753,6 +5762,42 @@ are all well-formed.
 
     3. Returns an operation state `op_state` that contains `op_state2`. When `execution::start(op_state)` is called, calls `execution::start(op_state2)`.
 
+    4. Given subexpressions `s2` and `e`, where `s2` is a sender returned from `into_variant` or a copy of such, let `S2` be `decltype((s2))` and `E` be `decltype((e))`.
+        Let <code><i>into-variant-set-value</i></code> be the class template:
+
+        <pre highlight=c++>
+            template &lt;class S, class E>
+            struct <i>into-variant-set-value</i> {
+              template &lt;class ...Args>
+              using <i>apply</i> = set_value_t(<i>into-variant-type</i>&lt;S, E>);
+            };
+        </pre>
+
+        Let <code><i>into-variant-is-nothrow</i></code> be the class template:
+
+        <pre highlight=c++>
+            template &lt;class S, class E>
+            struct <i>into-variant-is-nothrow</i> {
+              template &lt;class... Args>
+                using <i>apply</i> = noexcept(
+                  <i>into-variant-type</i>&lt;S, E>(<i>decayed-tuple</i>&lt;Args>(declval<Args>()...)));
```

Comment:
Is it a problem if this hard errors though? If this would hard error, then one of the calls to `set_value` would hard error, and the usage of `into_variant` is incorrect in the first place, no?